src/select.tsx

```diff
@@ -52,7 +52,8 @@ const useStyles = createUseStyles((theme) => ({
         left: 12 + 1, // 12 for select paddingLeft and +1 for border
         right: 48 + 1, // 48 for icon and +1 for border
         overflow: 'hidden',
-        top: ({label}) => (label ? 27 : 17),
+        top: ({label}) => (label ? 26 : 16),
```

Comment:
It's better than before but I can still see that the text is clipped :/ 

On the other hand, space between label and input text is too small compare with the standard input
![image](https://user-images.githubusercontent.com/6722153/155378971-6dd0c2f7-90dd-4f88-acbd-19d598ae7954.png)
![image](https://user-images.githubusercontent.com/6722153/155378998-051bf860-6fc6-465f-8318-90d1b70f4afc.png)